documentation/asciidoc/topics/proc_creating_caches_cli.adoc

```diff
@@ -1,71 +1,64 @@
 [id='creating_caches_cli-{context}']
 = Creating Caches with {brandname} CLI
-Use the command line interface (CLI) to create caches on your {brandname}
-cluster.
+Add cache configurations to your {brandname} cluster with the CLI.
```

Comment:
Just "Add caches to ..."?

The procedure detailed below is how to define and create a cache, not just a configuration.